docs/content/installation/csi_ug_install_operator_github.md

```diff
@@ -0,0 +1,59 @@
+# Installing the driver with GitHub
+
+The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
+
+Use the following steps to install the operator and driver, with [GitHub](https://github.com/IBM/ibm-block-csi-operator) (github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Before you begin, you may need to create a user-defined namespace. Create the project namespace, using the `kubectl create ns <namespace>` command.
+
+1.  Install the operator.
+
+    1. Download the manifest from GitHub.
```

Comment:
I noticed this [becomes i.](https://github.com/IBM/ibm-block-csi-driver/blob/56ea3ca6f02d8d0efa56fb3683ac5a73517f3cd3/docs/content/installation/csi_ug_install_operator_github.md) on GitHub, and becomes a. in ibm docs - could we align it to be the same?

Reply:
this is just how it transforms to IBM Docs and how it shows up in github. each one has it's own rendering of the MD files.